pdns/recursordist/Makefile.am

```diff
@@ -517,6 +517,9 @@ endif
 if HAVE_SYSTEMD
 pdns-recursor.service: pdns-recursor.service.in
 	$(AM_V_GEN)sed -e 's![@]sbindir[@]!$(sbindir)!' -e 's![@]service_user[@]!$(service_user)!' -e 's![@]service_group[@]!$(service_group)!' < $< > $@
+if !HAVE_SYSTEMD_WITH_RUNTIME_DIR_ENV
+	$(AM_V_GEN)sed -e 's!/pdns_recursor!& --socket-dir=/var/run/pdns-recursor-%i!' -i $@
```

Comment:
Should this get the same treatment as auth and use `%t` + drop `%-i` from the single instance.

Reply:
yes! I totally missed the rec